core/primitives/src/runtime/parameter_table.rs

```diff
@@ -33,6 +33,14 @@ impl ParameterValue {
         }
     }
 
+    fn as_u128(&self) -> Option<u128> {
```

Comment:
Is there a reason these conversions aren’t utilizing the `TryFrom` trait? It would let you to return descriptive errors too, rather than having to come up with some after the fact.